### DISCUSSION TITLE
Channel Topics

### DISCUSSION THREAD

**Discussion by ewainy** [ewainy/UkDevs#2] [Ideas]
Hey Friends! 💖

Just wanted to open a discussion where you can share your ideas for what channels you would like to see on the discord!

Do you want more support type channels with tech/career/content? 

Do you want more hobby channels? 

Let me know 😊

**Comment by conermurphy** [comment]
What do people think of a content share channel?

E.g. someone makes a blog post and wants to share the link for others to check out and share. Or, a product release for people to check out.

**Reply by ewainy** [reply]
We absolutely should have this!! It will be much easier to have a dedicated channel to share stuff.

**Comment by SamuelJForrest** [comment]
What do people think of a collaboration channel? 

Thought it might be a nice idea if anyone ever wanted to work on personal projects together?

**Reply by ewainy** [reply]
I love this idea Sam!! I'd wanted something like that & thought of arranging it on Github, hadn't even thought to introduce a channel for it! 😅
